### PR TITLE
Fix the unit tests for v1alpha2

### PR DESCRIPTION
**Pull request by jzp1025**
Fix the existing unit tests for v1alpha2 

This change is [Reviewable](https://reviewable.io/reviews/kubeflow/mxnet-operator/10)

### CODE REVIEW

**Comment by gaocegege** [comment]
Is it ready to review?

**Comment by jzp1025** [comment]
Yes , I have run through the tests and looks good .

File: pkg/controller.v2/mxnet/controller_test.go [hunk 1]
```diff
@@ -68,8 +68,8 @@ func newMXController(
 func TestNormalPath(t *testing.T) {
 	testCases := map[string]struct {
 		scheduler int
-		server int
                 worker int
```
Comment (gaocegege) [inline pkg/controller.v2/mxnet/controller_test.go]: The indent is not gofmted

**Comment by gaocegege** [comment]
@jlewi Should me enable Travis CI for mxnet operator or use prow directly?

**Comment by gaocegege** [comment]
/lgtm
/approve
/hold

@jzp1025 
Thanks for your contribution! While we need to decide whether to use Travis CI, thus I let tide hold the PR

**Comment by jzp1025** [comment]
Any progress in this PR ?
Thanks !
@gaocegege

**Comment by gaocegege** [comment]
/hold cancel
We need @suleisl2000 's approval.

**Pull request merged**